models/aether_lat-lon/readme.rst

```diff
@@ -0,0 +1,191 @@
+   even though electron temperature is.
+   They may be available in the history files.
+      
+
+model_nml
+.........
+
+template_file  
+   = 'if other than filter_input_0001.nc'
+
+variables
+   Each field to be included in the state vector requires 5 descriptors:
+   
+      1) field name (transformed to CF-compliant)
+      #) DART "quantity" to be associated with the field
+      #) min value
+      #) max value
+      #) update the field in the restart file? {UPDATE,NO_COPY_BACK}
+
+   The field names listed in ``variables`` must be the *transformed* names,
+   as found in the filter_input.nc files (see :ref:`Usage`).  
+   In general the transformation does the following
+   
+      - Remove all '\\', '(', and ')'
+      - Replace blanks with underscores
+      - Replace '+' with 'pos' and '-' with 'neg'
+      - For ions, move the ion name from the end to the beginning.
+   
+   For example 'velocity_parallel_east\\ \\(O+_2D\\)' becomes 'Opos_2D_velocity_parallel_east'.
+   
+   The DART QTY associated with each variable is an open question,
+   depending on the forward operators required for the available observations
+   and on the scientific objective.   The default choices are not necessarily correct
+   for your assimilation.  For the fields identified as most important
+   in early Aether assimilation experiments, these are the defaults::
+==============   ====================
+variable         quantity (kind)
+==============   ====================
+Temperature      QTY_TEMPERATURE
+velocity_east    QTY_U_WIND_COMPONENT
+velocity_north   QTY_V_WIND_COMPONENT
+Opos             QTY_DENSITY_ION_OP
+O2pos            QTY_DENSITY_ION_O2P
+N2pos            QTY_DENSITY_ION_N2P
+O2pos_2D         QTY_DENSITY_ION_O2DP
+O2pos_2P         QTY_DENSITY_ION_O2PP
+==============   ====================
+      
+   Some variables could have one of several QTYs associated with them.  
+   For example, the variable 'Opos_velocity_parallel_up'
+   could potentially have these existing QTYs associated with it::
+     - QTY_VELOCITY_W 
+     - QTY_VELOCITY_W_ION 
+     - QTY_VERTICAL_VELOCITY
+   It's possible that several variables could have the same QTY.
+   A third possibility is that the experiment may require the creation of a new QTY.
+   The example above may require something like QTY_VEL_PARALLEL_VERT_OP.
+
+.. WARNING:: 
+   The size of these parameters may be limited to 31 characters (``types_mod.f90``)
+
+time_step_days, time_step_seconds
+   = 0, 3600  The hindcast period between assimilations.
+
+.. _Usage:
+
+Usage
+-----
+
+To test the transformation of files for member 0:
+::
+
+> cd {aether_restart_dirname}
+> mkdir Orig
+> cp *m0000* Orig/
+> ./aether_to_dart  0
+> cp filter_input_0001.nc filter_output_0001.nc
+> ./dart_to_aether  0
+
+| The filter\_ files will contain the CF-compliant field names which must be used in model_nml:variables.
+| Compare the modified Aether restart files with those in Orig.
+.. NOTE:
```

Comment:
The note is not showing up in the documentation:  https://dart-documentation--635.org.readthedocs.build/en/635/models/aether_lat-lon/readme.html#usage
```suggestion

.. NOTE::
```